services.pug

```diff
@@ -108,7 +108,7 @@ block content
     .container
       .row
         .col-md-4(style="position: relative;")
-          .integration(style="height: 600px; background-color: #fff; border: solid 1px #ccc; position: relative; top: -100px; width: 100%; border-radius: 4px; box-shadow: 5px 10px 20px 5px #144199;")
+          .integration(style="height: 600px; background-color: #fff; border: solid 1px #ccc; position: relative; top: -100px; width: 100%; border-radius: 4px;box-shadow: 5px 10px 20px 5px #144199;")
```

Comment:
would be nice if you could keep the space " " between 4px;box-shadow => 4px; box-shadow